tests/tester_csv_writer.py

```diff
@@ -11,22 +12,34 @@ class TesterCsvWriter(TestCase):
     _measurement = None
     _host = None
     _database = None
-    host = 'ec2-34-211-57-101.us-west-2.compute.amazonaws.com'
-    port = 8086
-    username = 'root'
-    password = 'root'
+    host = db_config.host
+    port = db_config.port
+    username = db_config.username
+    password = db_config.password
     database = 'andrew_test_db'
 
-    def init(self):
+    def setUp(self):
         test_csv_write = CsvWriter(self.host, self.port, self.username, self.password, self.database)
-        self.assertNotEqual(0, test_csv_write._client,"Class generated properly")
+        self.assertNotEqual(0, test_csv_write._client, "Class generated properly")
 
-    def csv_file_to_data(self):
+    def test_csv_file_to_data(self):
         data_return_path = RESOURCES_DIR + "/" + 'temp2.csv'
         initial_data_path = RESOURCES_DIR + "/" + 'temp.csv'
         test_csv_write3 = CsvWriter(self.host, self.port, self.username, self.password, self.database)
         test_csv_write3.csv_file_to_db()
         test_csv_write3.data_to_csv_file('select * from per15min', new_csv_file_name=data_return_path)
-        self.assertTrue(filecmp.cmp(data_return_path, initial_data_path, shallow=False),
+        self.assertTrue(self.compare_csv_files(data_return_path, initial_data_path),
                         "Integration test failed, file is not the same")
         os.remove(data_return_path)
+
```

Comment:
Is the fileCompare not working. Why did you choose to do this?

Reply:
I figured that it throws the error in any case, so I decided to manually check for the lines. In the end it seems that there was a blank line in the new csv file.

Reply:
Okay. should be fine